### ISSUE TITLE
Stored Object: paper:openreview.BJLa

### ISSUE THREAD

**Issue by yanaiela** [yanaiela/papers-feed#1279]
{
  "sourceId": "openreview",
  "paperId": "BJLa",
  "url": "https://openreview.net/forum?id=BJLa_ZC9",
  "title": "Layer Normalization\ud83d\udcdd",
  "authors": "Jimmy Lei Ba, Jamie Ryan Kiros, Geoffrey E. Hinton",
  "abstract": "Training state-of-the-art, deep neural networks is computationally expensive. One way to reduce the training time is to normalize the activities of the neurons. A recently introduced technique called batch normalization uses the distribution of the summed input to a neuron over a mini-batch of training cases to compute a mean and variance which are then used to normalize the summed input to that neuron on each training case. This significantly reduces the training time in feed-forward neural networks. However, the effect of batch normalization is dependent on the mini-batch size and it is not obvious how to apply it to recurrent neural networks. In this paper, we transpose batch normalization into layer normalization by computing the mean and variance used for normalization from all of the summed inputs to the neurons in a layer on a single training case. Like batch normalization, we also give each neuron its own adaptive bias and gain which are applied after the normalization but before the non-linearity. Unlike batch normalization, layer normalization performs exactly the same computation at training and test times. It is also straightforward to apply to recurrent neural networks by computing the normalization statistics separately at each time step. Layer normalization is very effective at stabilizing the hidden state dynamics in recurrent networks. Empirically, we show that layer normalization can substantially reduce the training time compared with previously published techniques.",
  "timestamp": "2026-01-29T08:01:09.057Z",
  "rating": "novote",
  "publishedDate": "",
  "tags": [],
  "doi": "",
  "journalName": "",
  "sourceType": "url"
}

**Comment by yanaiela** [comment]
{
  "_data": {
    "sourceId": "openreview",
    "paperId": "BJLa",
    "url": "https://openreview.net/forum?id=BJLa_ZC9",
    "title": "Layer Normalization📝",
    "authors": "Jimmy Lei Ba, Jamie Ryan Kiros, Geoffrey E. Hinton",
    "abstract": "Training state-of-the-art, deep neural networks is computationally expensive. One way to reduce the training time is to normalize the activities of the neurons. A recently introduced technique called batch normalization uses the distribution of the summed input to a neuron over a mini-batch of training cases to compute a mean and variance which are then used to normalize the summed input to that neuron on each training case. This significantly reduces the training time in feed-forward neural networks. However, the effect of batch normalization is dependent on the mini-batch size and it is not obvious how to apply it to recurrent neural networks. In this paper, we transpose batch normalization into layer normalization by computing the mean and variance used for normalization from all of the summed inputs to the neurons in a layer on a single training case. Like batch normalization, we also give each neuron its own adaptive bias and gain which are applied after the normalization but before the non-linearity. Unlike batch normalization, layer normalization performs exactly the same computation at training and test times. It is also straightforward to apply to recurrent neural networks by computing the normalization statistics separately at each time step. Layer normalization is very effective at stabilizing the hidden state dynamics in recurrent networks. Empirically, we show that layer normalization can substantially reduce the training time compared with previously published techniques.",
    "timestamp": "2026-01-29T08:01:09.057Z",
    "rating": "novote",
    "publishedDate": "",
    "tags": [],
    "doi": "",
    "journalName": "",
    "sourceType": "url"
  },
  "_meta": {
    "client_version": "0.11.1",
    "timestamp": "2026-01-29T08:01:09.696Z",
    "update_mode": "append",
    "issue_number": 1279
  },
  "type": "initial_state"
}
